Cargo.toml

```diff
@@ -118,11 +118,11 @@ serde_json = "1.0.116"
 smallvec = "1"
 static_assertions = "1.1.0"
 thiserror = "1"
-wgpu = { version = "0.20.0", path = "./wgpu" }
-wgpu-core = { version = "0.21.0", path = "./wgpu-core" }
+wgpu = { version = "0.20.2", path = "./wgpu" }
+wgpu-core = { version = "0.21.1", path = "./wgpu-core" }
 wgpu-example = { version = "0.20.0", path = "./examples/common" }
-wgpu-macros = { version = "0.20.0", path = "./wgpu-macros" }
-wgpu-test = { version = "0.20.0", path = "./tests" }
+wgpu-macros = { version = "0.20.2", path = "./wgpu-macros" }
+wgpu-test = { version = "0.20.2", path = "./tests" }
```

Comment:
In general, unless we want to require people update a dependency, we shouldn't bump the version of the _dependency_.

Reply:
Point taken with auxiliary crates that haven't changed. `wgpu`. I reverted to previous contents on this hunk specifically. Looking good now?

But…wait, thought. Don't we _want_ to force `wgpu` users to upgrade to the fix motivating this release?I noticed wasn't updated in the 0.20.1 releaese.